lib/index.js

```diff
@@ -159,7 +164,7 @@ const register = function (server, options = {}) {
                 return h.continue;
             }
 
-            logger[defaultLevel]('request received', request.info.id, additionalFields);
+            logger[defaultLevel](`${request.method.toUpperCase()} ${request.path} request(${request.info.id}) received`, undefined, additionalFields);
```

Comment:
I think having at least the request id broken-out into `data` will be useful for consumers (rather than having to dig into the message string to get it).

Reply:
Something like: `{ ...additionalFields, id: request.info.id }` ?